src/zope/app/wsgi/testlayer.py

```diff
@@ -184,7 +187,8 @@ def getBody(self):
 
     def getOutput(self):
         status = self.response.status
-        status = status.encode('latin1') if not isinstance(status, bytes) else status
+        status = status.encode('latin1') if not isinstance(
+            status, bytes) else status
```

Comment:
Very minor, but I believe this is a one-liner for one reason only: to trick branchless coverage tracking into marking the statement as covered on both Python 2 and Python 3.

Given that we're now enabling branch coverage tracking everywhere we can stop being clever and write
```suggestion
        if not isinstance(status, bytes):
            status = status.encode('latin1')
```